smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -228,6 +243,35 @@ contract MixinKeys is
     return _ownerOf[_tokenId];
   }
 
+  /**
+  * @notice Get the key manager for the given tokenId
+  * @return The address of the manager for the given key
+  */
+  function getKeyManagerOf(
```

Comment:
In general do we prefer getters like this or public variables?  We should aim to pick one for consistency.

Reply:
yup. I think I should make this a public variable.

Reply:
I lean that way too. But a benefit of using getters instead is we can make variables private and therefore any use by other mixins is known to be contained to functions inside that contract.

Reply:
I think we had already migrated a handful of getters to public variables a ways back, so maybe go that way for consistency... 
one thing I should check is the size-diff between a public variable and a public getter function. NOte that if we think the benefits to using private vars outweighs the (assumed) size penalty I'm not against going that route.

Reply:
I'll open a new issue - seems drafting a style guide for questions like this would be useful.